pkg/exemplars/proxy_test.go

```diff
@@ -5,12 +5,17 @@ package exemplars
 
 import (
 	"context"
+	"fmt"
 	"io"
 	"os"
 	"reflect"
 	"sync"
 	"testing"
 
+	"github.com/prometheus/prometheus/promql/parser"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
```

Comment:
Can we group the imports?

Reply:
I keep getting ungrouped imports. Do you know why `goimports` doesn't sort them>

Reply:
Probably try something else like `gci`. `goimports` doesn't handle some cases well if I remember correctly

Reply:
Thanks! Should be sorted and grouped now.

Reply:
I guess it still needs some changes. We need to group thanos in the last section.

Reply:
I see, `goimports  -l -w -local github.com/thanos-io/thanos pkg/` seems to have done the trick.

Reply:
That's good to know!